### ISSUE TITLE
SCURLClassLoaderTests_0 failed with Unhandled exception on aarch64_linux_mixed

### ISSUE THREAD

**Issue by llxia** [eclipse/openj9#11965]
Internal Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/7

```
02:29:55.627  ===============================================
02:29:55.627  Running test SCURLClassLoaderTests_0 ...
02:29:55.627  ===============================================
02:29:55.627  SCURLClassLoaderTests_0 Start Time: Fri Feb 12 07:53:38 2021 Epoch Time (ms): 1613116418559
02:29:55.627  "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdkbinary/j2sdk-image/bin/java" -Xshareclasses:destroyAll; "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdkbinary/j2sdk-image/bin/java" -Xshareclasses:groupAccess,destroyAll; echo "cache cleanup done";
02:29:55.627  JVMSHRC005I No shared class caches available
02:29:55.627  JVMSHRC005I No shared class caches available
02:29:55.627  cache cleanup done
02:29:55.627  variation: Mode110
02:29:55.627  JVM_OPTIONS:  -Xjit -Xgcpolicy:gencon -Xnocompressedrefs 
02:29:55.627  { itercnt=1; \
02:29:55.627  mkdir -p "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdk-tests/TKG/../TKG/output_16131135515881/SCURLClassLoaderTests_0"; \
02:29:55.627  cd "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdk-tests/TKG/../TKG/output_16131135515881/SCURLClassLoaderTests_0"; \
02:29:55.627  cp -r "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdk-tests/TKG/../../jvmtest/functional/cmdLineTests/URLClassLoaderTests/URLClassLoaderTests.jar" .; \
02:29:55.627  "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdkbinary/j2sdk-image/bin/jar" -xf URLClassLoaderTests.jar; \
02:29:55.627  "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdkbinary/j2sdk-image/bin/java"  -Xjit -Xgcpolicy:gencon -Xnocompressedrefs  -DJAVA_HOME='/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdkbinary/j2sdk-image' -DPATHSEP="/" -DRUN_SCRIPT=sh -DSCRIPT_SUFFIX=.sh -DEXECUTABLE_SUFFIX= -DJAVA_EXE='"/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdkbinary/j2sdk-image/bin/java"  -Xjit -Xgcpolicy:gencon -Xnocompressedrefs ' -DCPDL=":" -DSCMODE=210 -DTEST_JVM_OPTIONS=" -Xjit -Xgcpolicy:gencon -Xnocompressedrefs " \
02:29:55.627  -jar "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdk-tests/TKG/../../jvmtest/functional/cmdline_options_tester/cmdlinetester.jar" \
02:29:55.627  -config "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdk-tests/TKG/../../jvmtest/functional/cmdLineTests/URLClassLoaderTests/URLClassLoaderTests.xml" -xids all,linux_aarch64_mr,latest,11 -xlist "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdk-tests/TKG/../../jvmtest/functional/cmdLineTests/URLClassLoaderTests/exclude.xml" \
02:29:55.627  -nonZeroExitWhenError \
02:29:55.627  -outputLimit 300; \
02:29:55.627  if [ $? -eq 0 ] ; then echo ""; echo "SCURLClassLoaderTests_0""_PASSED"; echo ""; cd /home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdk-tests/TKG/..; rm -f -r "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdk-tests/TKG/../TKG/output_16131135515881/SCURLClassLoaderTests_0"; else echo ""; echo "SCURLClassLoaderTests_0""_FAILED"; echo ""; fi; } 2>&1 | tee -a "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdk-tests/TKG/../TKG/output_16131135515881/TestTargetResult";
02:29:56.788  Parsing exclude list...
02:29:56.788    add excludes for [all]
02:29:56.788    add excludes for [linux_aarch64_mr]
02:29:56.788    add excludes for [latest]
02:29:56.788    add excludes for [11]
02:29:56.788  DONE in 0 ms.
02:29:56.788  *** Starting test suite: Shared Classes URLClassLoaderTests Suite ***
02:29:56.788   
02:29:56.788  #######################################################
02:29:56.788  Running tests in mode 210 with command line options:  
02:29:56.788  #######################################################
02:29:56.788   
02:29:56.788  Executing command: "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdkbinary/j2sdk-image/bin/java"  -Xjit -Xgcpolicy:gencon -Xnocompressedrefs  -Xshareclasses:destroyAll
02:29:56.788  
02:29:56.788  Executing command: "/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdkbinary/j2sdk-image/bin/java"  -Xjit -Xgcpolicy:gencon -Xnocompressedrefs  -Xshareclasses:persistent,controlDir=.,name=invalidCacheName,destroyAll
02:29:56.788  
02:29:56.788  Testing: Sanity.BuildInitial
02:29:56.788  Unhandled exception
02:29:56.788  Type=Segmentation error vmState=0x00000000
02:29:56.788  J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
02:29:56.788  Handler1=0000FFFF7F7D38C0 Handler2=0000FFFF7F74B3E8 InaccessibleAddress=00000000782B6360
02:29:56.788  R0=0000FFFF7813D600 R1=0000FFFE6167956C R2=0000FFFE61679870 R3=0000FFFF80651B50
02:29:56.788  R4=0000000000000000 R5=0000FFFF78214D38 R6=0000FFFF78255028 R7=0000FFFE61679858
02:29:56.788  R8=0000FFFE61679558 R9=0000000010000000 R10=0000FFFE6167956C R11=0000000000000000
02:29:56.788  R12=0000FFFF77EFB8B8 R13=0061006A004C0029 R14=00000000000D75B8 R15=0000FFFF80651CD0
02:29:56.788  R16=0000FFFF7EAA7AE8 R17=0000FFFF80707F80 R18=0000000000000020 R19=0000FFFF7813D600
02:29:56.788  R20=00000000782B6300 R21=0000FFFE7E14E4D0 R22=0000FFFE7E017618 R23=0000FFFF77EFBD00
02:29:56.788  R24=00000000F9F3B8B0 R25=000000000000001A R26=0000FFFF7F9633B0 R27=0000FFFF7F947CD8
02:29:56.788  R28=0000FFFF7F9633B0 R29=0000FFFF80651A80 R30=0000FFFF7EE92A64 R31=0000FFFF80651A80
02:29:56.788  PC=0000FFFF7EE9E9AC SP=0000FFFF80651A80 PSTATE=0000000080000000
02:29:56.788  V0 0000ffff78214d48 (f: 2015448448.000000, d: 1.390660e-309)
02:29:56.788  V1 0000000048da6220 (f: 1222271488.000000, d: 6.038824e-315)
02:29:56.788  V2 000000003f800000 (f: 1065353216.000000, d: 5.263544e-315)
02:29:56.788  V3 bfe02da414c016bf (f: 348133056.000000, d: -5.055714e-01)
02:29:56.788  V4 bfd00ea348b88334 (f: 1220051712.000000, d: -2.508934e-01)
02:29:56.788  V5 3fd5575b0be00b6a (f: 199232368.000000, d: 3.334568e-01)
02:29:56.788  V6 bfdffffef20a4123 (f: 4060758272.000000, d: -4.999997e-01)
02:29:56.788  V7 3fe62e42fefa39ef (f: 4277811712.000000, d: 6.931472e-01)
02:29:56.788  V8 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.788  V9 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.788  V10 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.788  V11 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.788  V12 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.788  V13 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.788  V14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.788  V15 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.788  V16 bff0000000000000 (f: 0.000000, d: -1.000000e+00)
02:29:56.788  V17 fd00000000000000 (f: 0.000000, d: -1.277338e+294)
02:29:56.789  V18 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.789  V19 0000ffff80650b10 (f: 2154105600.000000, d: 1.390661e-309)
02:29:56.789  V20 0000ffff80650b00 (f: 2154105600.000000, d: 1.390661e-309)
02:29:56.789  V21 0000ffff80650af0 (f: 2154105600.000000, d: 1.390661e-309)
02:29:56.789  V22 0000ffff80650ae0 (f: 2154105600.000000, d: 1.390661e-309)
02:29:56.789  V23 00000000000000f0 (f: 240.000000, d: 1.185758e-321)
02:29:56.789  V24 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.789  V25 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.789  V26 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.789  V27 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.789  V28 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.789  V29 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.789  V30 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.789  V31 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:29:56.789  Module=/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdkbinary/j2sdk-image/lib/default/libj9jit29.so
02:29:56.789  Module_base_address=0000FFFF7E742000
02:29:56.789  Target=2_90_20210212_30 (Linux 4.15.0-132-generic)
02:29:56.789  CPU=aarch64 (8 logical CPUs) (0x3e8127000 RAM)
02:29:56.789  ----------- Stack Backtrace -----------
02:29:56.789  (0x0000FFFF7F748040 [libj9prt29.so+0x26040])
02:29:56.789  ---------------------------------------
02:29:56.789  JVMDUMP039I Processing dump event "gpf", detail "" at 2021/02/12 07:53:40 - please wait.
02:29:56.789  JVMDUMP032I JVM requested System dump using '/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdk-tests/TKG/output_16131135515881/SCURLClassLoaderTests_0/core.20210212.075340.32251.0001.dmp' in response to an event
02:30:00.428  JVMPORT030W /proc/sys/kernel/core_pattern setting "|/usr/share/apport/apport %p %s %c %d %P %E" specifies that the core dump is to be piped to an external program.  Attempting to rename either core or core.32304.
02:30:00.428  
02:30:00.428  JVMDUMP010I System dump written to /home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdk-tests/TKG/output_16131135515881/SCURLClassLoaderTests_0/core.20210212.075340.32251.0001.dmp
02:30:00.428  JVMDUMP032I JVM requested Java dump using '/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdk-tests/TKG/output_16131135515881/SCURLClassLoaderTests_0/javacore.20210212.075340.32251.0002.txt' in response to an event
02:30:01.599  JVMDUMP010I Java dump written to /home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdk-tests/TKG/output_16131135515881/SCURLClassLoaderTests_0/javacore.20210212.075340.32251.0002.txt
02:30:01.599  Unhandled exception
02:30:01.599  Type=Segmentation error vmState=0x0002000f
02:30:01.599  J9Generic_Signal_Number=00000018 Signal_Number=0000000b Error_Value=00000000 Signal_Code=00000001
02:30:01.599  Handler1=0000FFFF7F7D3C58 Handler2=0000FFFF7F74B3E8 InaccessibleAddress=0000000000000000
02:30:01.599  R0=0000FFFF77EFBD00 R1=0000FFFF77EFBD00 R2=0000000000000000 R3=0000FFFF7813D600
02:30:01.599  R4=FFFFFFFFE0BC0517 R5=0000FFFE7E017618 R6=0000000000000000 R7=0000000000000000
02:30:01.599  R8=0000000000000009 R9=0000000000000003 R10=0000000000000004 R11=00000000FFFFFFFC
02:30:01.599  R12=000000000000000A R13=0000FFFE600BC738 R14=00000000000000E4 R15=0000FFFF7F93FBB0
02:30:01.599  R16=0000FFFF7F9600F0 R17=0000FFFF80708800 R18=0000000000000004 R19=FFFFFFFFE0BC0517
02:30:01.599  R20=0000FFFE7E017618 R21=0000FFFE7E017600 R22=0000FFFE7E017618 R23=0000FFFF78214D50
02:30:01.599  R24=0000000000000000 R25=0000000000000000 R26=0000FFFF7E731DE0 R27=0000FFFE600BE1F0
02:30:01.599  R28=0000FFFE600BD060 R29=0000FFFE600BCF20 R30=0000FFFF7F813520 R31=0000FFFE600BCF10
02:30:01.599  PC=0000FFFF7F812A90 SP=0000FFFE600BCF10 PSTATE=0000000020000000
02:30:01.599  V0 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V1 408f400000000000 (f: 0.000000, d: 1.000000e+03)
02:30:01.599  V2 412e848000000000 (f: 0.000000, d: 1.000000e+06)
02:30:01.599  V3 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V4 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V5 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V6 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V7 3fe62e42fefa39ef (f: 4277811712.000000, d: 6.931472e-01)
02:30:01.599  V8 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V9 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V10 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V11 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V12 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V13 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V14 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V15 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V16 bff0000000000000 (f: 0.000000, d: -1.000000e+00)
02:30:01.599  V17 fcfcfe00feff0000 (f: 4278124544.000000, d: -1.157276e+294)
02:30:01.599  V18 0000ffff80650b20 (f: 2154105600.000000, d: 1.390661e-309)
02:30:01.599  V19 0000ffff80650b10 (f: 2154105600.000000, d: 1.390661e-309)
02:30:01.599  V20 0000ffff80650b00 (f: 2154105600.000000, d: 1.390661e-309)
02:30:01.599  V21 0000ffff80650af0 (f: 2154105600.000000, d: 1.390661e-309)
02:30:01.599  V22 0000ffff80650ae0 (f: 2154105600.000000, d: 1.390661e-309)
02:30:01.599  V23 00000000000000f0 (f: 240.000000, d: 1.185758e-321)
02:30:01.599  V24 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V25 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V26 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V27 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V28 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V29 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V30 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  V31 0000000000000000 (f: 0.000000, d: 0.000000e+00)
02:30:01.599  Module=/home/jenkins/workspace/Test_openjdk11_j9_sanity.functional_aarch64_linux_mixed/openjdkbinary/j2sdk-image/lib/default/libj9vm29.so
02:30:01.599  Module_base_address=0000FFFF7F7A0000
02:30:01.599  Target=2_90_20210212_30 (Linux 4.15.0-132-generic)
02:30:01.599  CPU=aarch64 (8 logical CPUs) (0x3e8127000 RAM)
02:30:01.599  ----------- Stack Backtrace -----------
02:30:01.599  (0x0000FFFF7F748040 [libj9prt29.so+0x26040])
02:30:01.599  ---------------------------------------
02:30:01.599  JVMDUMP039I Processing dump event "gpf", detail "" at 2021/02/12 07:53:44 - please wait.
```

This also hangs the test build.

**Comment by pshipton** [comment]
@knn-k

**Comment by pshipton** [comment]
Note this is -Xnocompressedrefs

**Comment by llxia** [comment]
auto exclude test SCURLClassLoaderTests_0 plat=.*aarch.*

**Comment by knn-k** [comment]
Issue #11977 AArch64: Support mixed refs

**Comment by knn-k** [comment]
I merged #16495, and removed the "test excluded" label from this issue.
